docs/source/community/contributing.rst

```diff
@@ -2,100 +2,95 @@
 Contributing
 ------------
 
-==================
-Ways to Contribute
-==================
+==========================================
+Ways to Contribute to Hyperledger Sawtooth
+==========================================
 
-Contributions by the community help grow and optimize the capabilities of
-Hyperledger Sawtooth, and are the most effective method of having a positive
-impact on the project.
+Contributions from the development community help improve the capabilities of
+Hyperledger Sawtooth. These contributions are the most effective way to
+make a positive impact on the project.
 
-**Different ways you can contribute**
+Ways you can contribute:
 
-* Bugs or Issues (issues or defects found when working with Sawtooth)
-* Core Features & Enhancements (expanded capabilities or optimization)
-* Arcade Features (games that demonstrate Sawtooth such as Tic-Tac-Toe
-  and Battleship)
-* New or Enhanced Documentation (improve existing documentation or create new)
-* Testing Events and Results (functional, performance or scalability)
+* Bugs or issues: Report problems or defects found when working with Sawtooth
+* Core features and enhancements: Provide expanded capabilities or optimizations
+* Arcade features: Add games that demonstrate Sawtooth; see the existing
+  games XO (tic-tac-toe) and Battleship
```

Comment:
Arcade is from a much earlier version of sawtooth. We still have some of these games but no longer an 'arcade' concept. May as well cull this as a suggestion. Instead, I think we are generally interested in example applications such as what TMobile contributed with the role based authentication application https://github.com/hyperledger/sawtooth-hyper-directory

Reply:
Thanks for the info. I'll delete the "Arcade features" line in this PR.  If new information is needed, it can go in a separate PR.